definitions/divorce/json/CaseEvent.json

```diff
@@ -1793,6 +1793,19 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "07/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "updateBulkCaseHearingDetails",
+    "Name": "Update case with court hearing",
+    "Description": "Update case with court hearing details",
+    "DisplayOrder": 21,
+    "PreConditionState(s)": "AwaitingPronouncement",
+    "PostConditionState": "AwaitingPronouncement",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/generate-document?templateId=FL-DIV-GNO-ENG-00020.docx&documentType=coe&filename=certificateOfEntitlement",
```

Comment:
the file name is defined in the calblack?? :(

Reply:
Yeah, so we can have just one endpoint for all generate doc and attach to case events, and have full control over the file parameters. The callback itself appends caseId to the passed in filename

Reply:
Oh I didn't notice that lol